tools/topology/sof-bdw-codec.m4

```diff
@@ -45,12 +45,27 @@ PIPELINE_PCM_ADD(sof/pipe-low-latency-capture.m4,
 	1000, 0, 0,
 	48000, 48000, 48000)
 
+#
+# DAI configuration
+#
+# SSP port 0 is our only pipeline DAI
+#
+
+# playback DAI is SSP0 using 2 periods
+# Buffers use s24le format, 1000us deadline on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-playback.m4,
+	1, SSP, 0, Codec,
+	PIPELINE_SOURCE_1, 2, s24le,
+	1000, 0, 0)
+
 # PCM Media Playback pipeline 3 on PCM 1 using max 2 channels of s32le.
 # 2000us deadline on core 0 with priority 1
 PIPELINE_PCM_ADD(sof/pipe-pcm-media.m4,
 	3, 1, 2, s32le,
-	2000, 1, 0,
-	8000, 96000, 48000)
+	1000, 0, 0,
+	8000, 96000, 48000,
+	0, PIPELINE_PLAYBACK_SCHED_COMP_1)
```

Comment:
why is this different from pipeline 1 in terms of scheduling?

Reply:
not sure, I follow the working m4 from cht. I think they all use the same DMA and maybe should follow the same schedule.